src/components/Form/formFunctions.ts

```diff
@@ -65,20 +66,23 @@ export const createSetter = <SpecificFields extends Fields>(
     };
 };
 
-export const getErrorType = (
+export const getErrorType = <SpecificFields extends Fields>(
     input: string,
+    key: keyof SpecificFields,
     required?: boolean,
     regex?: RegExp,
     additionalCondition?: (value: string) => boolean
 ): Errors => {
     if (input == "") {
         return required ? Errors.required : Errors.none;
     }
-
     if (
         (regex !== undefined && !input.match(regex)) ||
         (additionalCondition !== undefined && !additionalCondition(input))
     ) {
+        if (key === "deliveryInstructions") {
+            return Errors.deliveryInstructions;
+        }
         return Errors.invalid;
     }
```

Comment:
We might want to implement character limits elsewhere. Can this just be a super generic character limit error? rather than passing `key` as a prop we can just pass `maxCharacters?: number` then check the length inside